lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -90,6 +90,12 @@
       - Ignore dependencies, only install/upgrade the package itself
     default: false
     version_added: '2.1'
+  allow_multiple_versions:
+    description:
+      - Allow multiple versions of the same package to be installed
+    require: false
```

Comment:
Remove this line. The default behaviour is that parameters are not required, unless you enable this.